onnx/defs/math/defs.cc

```diff
@@ -1069,6 +1069,69 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetContextDependentFunctionBodyBuilder(BuildContextDependentFunctionBodyClip)
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+std::function<void(OpSchema&)>
+SoftmaxFamilyDocGenerator(const char* name, const char* description, const char* equation) {
+  return [=](OpSchema& schema) {
+    std::string doc;
+    POPULATE_OP_DOC_STR(doc = R"DOC(
+The operator computes the {description} values for the given input:
+
+ {equation}
+
+The "axis" attribute indicates the dimension along which {name}
+will be performed. The output tensor has the same shape
+and contains the {name} values of the corresponding input.
+)DOC";
+                        ReplaceAll(doc, "{name}", name);
```

Comment:
Indentation looks off

Reply:
For some reason this is what clangformat produced. I will try to nudge it a little

Reply:
Strange, but I have seen this before too, so it makes sense it is the formatter. May be the macro is confusing it.

Reply:
I haven't figured this out. Will try again later